scss/bitstyles/ui/sidebar.stories.mdx

```diff
@@ -3,22 +3,14 @@ import logoLarge from '../../../test/assets/images/logoLarge.svg';
 import logoSmall from '../../../test/assets/images/logoSmall.svg';
 import icons from '../../../assets/images/icons.svg';
 
-<Meta title="UI/Navigation/Sidebar" />
+<Meta title="Organisms/Sidebar" />
```

Comment:
This has the same name as the `.js` file with the sidebar stories in, so neither of them shows up in storybook’s navigation (names need to be unique so storybook knows which is meant). Let’s keep this as being in the UI layer, since it shows complete examples on how to use this component
```suggestion
<Meta title="UI/Navigation/Sidebar" />
```